tasks/stylus.js

```diff
@@ -154,6 +174,10 @@ module.exports = function(grunt) {
     }, function() {
       grunt.log.ok(filesCreatedCount + ' ' + grunt.util.pluralize(filesCreatedCount, 'file/files') + ' created.');
 
+      if (sourcemapsCreated) {
+        grunt.log.ok(sourcemapsCreated + ' ' + grunt.util.pluralize(sourcemapsCreated, 'sourcemap/sourcemaps') + ' created.');
+      }
```

Comment:
I added log above to have greater visibility on when sourcemaps are generated and that they're being generated correctly. Output example:

![screenshot 2016-08-03 17 01 20](https://cloud.githubusercontent.com/assets/1557348/17382307/384e6a7a-599e-11e6-8c80-4c2c0d6cd3a0.png)